ambry-replication/src/test/java/com.github.ambry.replication/ReplicationTest.java

```diff
@@ -2039,12 +2195,14 @@ public long sizeInBytes() {
     private List<ByteBuffer> buffersToReturn;
     private Map<PartitionId, List<MessageInfo>> infosToReturn;
     private Map<PartitionId, List<MessageMetadata>> messageMetadatasToReturn;
+    private Map<StoreKey, StoreKey> conversionMap;
     private ReplicaMetadataRequest metadataRequest;
     private GetRequest getRequest;
 
-    MockConnection(Host host, int maxSizeToReturn) {
+    MockConnection(Host host, int maxSizeToReturn, Map<StoreKey, StoreKey> conversionMap) {
```

Comment:
Instead of having one ctor whee most of the time you fill the last argument with `null`, create another ctor `MockConnection(Host host, int maxSizeToReturn)` that calls `this(host, maxSizeToReturn, null)`, and then replace all the calls for `new MockConnection(host, int, null)` with `new MockConnection(host, int)`

Reply:
Good point. Fixed.